nector.js.org

RajNandigalla opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

Hi,

Can you please approve this .

MattIPv4 reviewed (commented):
Do you have a link to where we can preview the content in advance?

RajNandigalla commented:
here you go 
https://nector-web-git-feat-blog-author.nraj.now.sh/editor @MattIPv4

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But you have to add the linked content.
I'll merge when the page has "resonable content".

RajNandigalla commented:
> I've already added your requested subdomain to JS.ORGs zonefile. But you have to add the linked content.
> I'll merge when the page has "resonable content".

I have updated the main Page , its a webpage builder 
https://nector-web-git-feat-blog-author.nraj.now.sh/
please approve @indus

indus commented:
Welcome https://nector.js.org
